docs/book/src/reference/envtest.md

```diff
@@ -183,6 +183,56 @@ expectedOwnerReference := v1.OwnerReference{
 }
 Expect(deployment.ObjectMeta.OwnerReferences).To(ContainElement(expectedOwnerReference))
 ```
+<aside class="warning">
```

Comment:
that is great. Just seem that the content is duplicated.
Could you please check that?

Reply:
Sure